apps/tasks/lib/components/user_bottom_sheet.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_theme/theme.dart';
+import 'package:provider/provider.dart';
+
+
+class UserBottomSheet extends StatefulWidget {
+   const UserBottomSheet({super.key});
+
+  @override
+  State<UserBottomSheet> createState() => _UserBottomSheetState();
+}
+
+class _UserBottomSheetState extends State<UserBottomSheet> {
+  final items = ["station", "station 1"];
+  String? selectedStation;
+
+  @override
+  Widget build(BuildContext context) {
+    final double height  = MediaQuery.of(context).size.height;
+    final double width = MediaQuery.of(context).size.width;
+
+    return Consumer<ThemeModel>(builder: (BuildContext context, ThemeModel themeNotifier, __){
+      return BottomSheet(
+        animationController: AnimationController(
+          vsync: Navigator.of(context),
+          duration: const Duration(milliseconds: 500),
+        ),
+        elevation: 10,
+        onClosing: () {},
+        enableDrag: true,
+        builder: (BuildContext ctx) => Container(
```

Comment:
Why are you using a stack? A normal `Column` wrapped in a `Padding` should do the same here and a Row for the close and logout buttons.

Also increase the padding between the clinic name and the dropdown